modular_zubbers/code/modules/cargo/packs/goodies.dm

```diff
@@ -74,3 +74,9 @@
 
 /datum/supply_pack/goody/thermal_single
 	special = FALSE
+
+/datum/supply_pack/goody/medkit_surgery
+	name = "Spare Surgical Medkit"
+	desc = "A spare high capacity medkit containing a set of surgical tools and basic supplies."
+	cost = PAYCHECK_CREW * 10
```

Comment:
```suggestion
	cost = PAYCHECK_CREW * 15
```

Reply:
This box has 12 slots and fits in a backpack. It is a premium item.

Reply:
yeah i guess it could be more expensive - can lower the price later if people complain